migrations/20220601201014_rename_campaign_report_to_irregularity/migration.sql

```diff
@@ -1,9 +1,5 @@
 /*
-  Warnings:
-
-  - You are about to drop the `campaign_reports` table. If the table is not empty, all the data it contains will be lost.
-  - You are about to drop the `report_files` table. If the table is not empty, all the data it contains will be lost.
-
+The below migration is written by hand because the default migration drops and recreates tables
```

Comment:
@quantum-grit Modifying already deployed migration changes it's file sha hash, so a warning will be shown in the logs when we deploy further migrations.

This is general info, no actions are needed.